packages/shared-components/src/viewmodel/tests/useCreateAutoDisposedViewModel.test.ts

```diff
@@ -0,0 +1,47 @@
+/*
+Copyright 2025 New Vector Ltd.
```

Comment:
I assume this probably should be Element Creations? (Also could do with a new line below)